package.nls.json

```diff
@@ -24,6 +24,7 @@
     "azureFunctions.deleteFunction": "Delete Function...",
     "azureFunctions.deleteFunctionApp": "Delete Function App...",
     "azureFunctions.deleteSlot": "Delete Slot...",
+    "azureFunctions.deployProject": "Deploy to Azure...",
```

Comment:
> Removed Create Function App... option in the local workspace view
This was replaced with just Deploy function to Azure... which includes a + Create new function app option as a quick pick when we prompt for the app

![image](https://github.com/microsoft/vscode-azurefunctions/assets/12476526/66d6c298-61fd-42fd-bbda-1589f324105f)

Is it supposed to be "Deploy to Azure..." or "Deploy function to Azure..."?

Reply:
Ah, I think we originally went with "Deploy function to Azure..." but after more discussion, just went with a more generic "Deploy to Azure..."

Reply:
We don't think we need to give more context to users on _what_ they are deploying?

Reply:
Yeah, I mean you're in the workspace part of the extension so I think it's kind of clear. That being said, I think it still prompts you for a workspace folder.

I'm thinking of adding a right-click context item on the local project node though that also has "Deploy to Azure..." on it.

Reply:
Well I was thinking that they might need to know that the command is for deploying Functions to Azure and not App Service or anything else. But maybe they should already know?

Reply:
Well, you click the Functions symbol to open the menu for the commands. I thought that was kind of the reason we had those buttons there to give them that context.

Reply:
Ok, that's fine